### ISSUE TITLE
The w in the toolbar

### ISSUE THREAD

**Issue by Dion-F** [JasonWibbo/HabboSwfOpenSource#3]
Dear,

I changed the W in the toolbar using flashdec but its still there -> http://prntscr.com/rqq6r8

**Comment by JasonWibbo** [comment]
May be HabboWindowManagerCom_bottom_bar_logo.png

**Comment by JasonWibbo** [comment]
https://github.com/JasonWibbo/HabboSwfOpenSource/blob/master/src/images/HabboWindowManagerCom_bottom_bar_logo.png

**Comment by Dion-F** [comment]
So i need to compile the SWF myself?

**Comment by JasonWibbo** [comment]
Yes, dowload Adobe Flex 4.6 SDK https://www.adobe.com/devnet/flex/flex-sdk-download.html and edit https://github.com/JasonWibbo/HabboSwfOpenSource/blob/master/compile.php
"C:\Users\Wibbo\Downloads\flex_sdk_4.6\bin\mxmlc.exe" for you link

**Comment by Dion-F** [comment]
Also is there a reason wibboemulator was deleted from github?

**Comment by Dion-F** [comment]
Error: cannot open' C:\HabboSwfOpenSource-master\HabboSwfOpenSource-master\src\Habbo.as'

**Comment by Dion-F** [comment]
http://prntscr.com/rqvpln

**Comment by JasonWibbo** [comment]
I have released the emulator in public.
Hm i do not really know

**Comment by Dion-F** [comment]
I think swf broken cus original source without script renaming i can compile
